pom.xml

```diff
@@ -117,6 +118,18 @@
       <artifactId>jnr-unixsocket-nodep</artifactId>
       <version>0.3.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>annotations</artifactId>
+      <version>3.0.0</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-step-api</artifactId>
+      <version>${workflow-jenkins-plugin.version}</version>
```

Comment:
@batmat they are adding hard dep to workflow-step-api in all plugins. Even credentials-binding. I got reply that they (CB) don't care :(